sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -100,92 +130,163 @@ ServiceBusReceiverAsyncClient receiver = new ServiceBusClientBuilder()
-
-* [Queue][queue_concept]: Allows for Sending and Receiving of messages, ordered first-in-first-out.  Often used for
-point-to-point communication.
+on which actual message transmission takes place. The namespace often serves as an application container:
 
-* [Topic][topic_concept]: As opposed to Queues, Topics are better suited to publish/subscribe scenarios.  A topic can
-be sent to, but requires a subscription, of which there can be multiple in parallel, to consume from.
-
-* [Subscription][subscription_concept]: The mechanism to consume from a Topic.  Each subscription is independent, and
-receaves a copy of each message sent to the topic.
+* A **[queue][queue_concept]** allows for the sending and receiving of messages, ordered first-in-first-out. It is often
+  used for point-to-point communication.
+* A **[topic][topic_concept]** is better suited to publisher and subscriber scenarios. A topic publishes messages to
+  _subscriptions_, of which, multiple can exist simultaneously.
+* A **[subscription][subscription_concept]** receives messages from a topic. Each subscription is independent and
+  receives a copy of the message sent to the topic.
 
 ## Examples
 
-### Create a sender or receiver using connection string
-
-The easiest means for doing so is to use a connection string, which is created automatically when creating an Service Bus
-namespace. If you aren't familiar with shared access policies in Azure, you may wish to follow the step-by-step guide to
-[get an Service Bus connection string][service_bus_connection_string].
+### Send messages
 
-Both the asynchronous and synchronous Service Bus sender and receiver clients can be created using
-`ServiceBusClientBuilder`.The examples are explained blow.
+You'll need to create an asynchronous [`ServiceBusSenderAsyncClient`][ServiceBusSenderAsyncClient] or a synchronous
+[`ServiceBusSenderClient`][ServiceBusSenderClient] to send messages. Each sender can send messages to either a queue or
+a topic.
 
-The snippet below creates an asynchronous Service Bus sender.
+The snippet below creates a synchronous `ServiceBusSender` to publish a message to a queue.
 
-<!-- embedme ./src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java#L23-L28 -->
+<!-- embedme ./src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java#L65-L74 -->
 ```java
-String connectionString = "<< CONNECTION STRING FOR THE SERVICE BUS NAMESPACE >>";
-ServiceBusSenderAsyncClient sender = new ServiceBusClientBuilder()
-    .connectionString(connectionString)
+ServiceBusSenderClient sender = new ServiceBusClientBuilder()
+    .connectionString("<< CONNECTION STRING FOR THE SERVICE BUS NAMESPACE >>")
     .sender()
     .queueName("<< QUEUE NAME >>")
-    .buildAsyncClient();
+    .buildClient();
+List<ServiceBusMessage> messages = Arrays.asList(
+    new ServiceBusMessage("Hello world".getBytes()).setMessageId("1"),
+    new ServiceBusMessage("Bonjour".getBytes()).setMessageId("2"));
```

Comment:
Do we want to include a `Charset` in the `getBytes` call?

Reply:
I believe we're considering doing a default string => utf8-bytes type conversion in all the libraries, so doesn't sound like a bad idea.

Reply:
I can specify it. Trying to decrease the length of characters. lol